q2_feature_table/tests/filter/test_filter_sequences.py

```diff
@@ -0,0 +1,102 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import unittest
+
+import qiime2
+import pandas as pd
+from pandas.util.testing import assert_series_equal
+
+from q2_feature_table import filter_seqs
+
+
+class FilterSeqsTests(unittest.TestCase):
+    def setUp(self):
+        self.seqs = pd.Series(['ACGT', 'GCTA', 'CCCC', 'TGTT'],
+                              index=['O1', 'O2', 'O3', 'O4'])
+        self.md_lite = pd.DataFrame(
+            [['A'], ['C'], ['G'], ['T']], index=['O1', 'O2', 'O3', 'O4'],
+            columns=['seq'])
+        md_full = pd.DataFrame(
+            [['foo', '1'], ['bar', '2'], ['baz', '3'], ['foo', '4']],
+            index=['O1', 'O2', 'O3', 'O4'], columns=['stuff', 'some_numbers'])
+        md_full.index.name = 'FeatureID'
+        self.md_full = qiime2.Metadata(md_full)
+
+    def assertFiltered(self, exp, md=None, exclude_ids=False,
```

Comment:
This name is kind of confusing to me, I really expect it to perform only a comparison (of some kind), not run the function that we're testing.

What if it was 
```python
def expect_after_filter(...):
```

or something like that?

Alternatively it looks like there's a fair bit of wiring to save 2 lines in practice, maybe we don't need this?

Reply:
I originally wrote the tests without this helper, in my opinion it was pretty clunky to try and read the tests and understand what was going one. I agree about the wiring, but I felt like the readability payoff was worth it.

Reply:
What about `filter_and_assertEqual`?